Makefile

```diff
@@ -71,10 +69,8 @@ kube-rc:
 	kubectl create -f ${RC}
 
 kube-clean:
-	kubectl delete rc deis-builder
-
-test:
-	@echo "Implement functional tests in _tests directory"
```

Comment:
we should retain this. It shows that we intend to add tests, and that `make test` is the way to run them all.